src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -22,6 +22,7 @@ import { groupBy, sortBy } from 'lodash';
 import React, { Fragment, useRef } from 'react';
 import useObservable from 'react-use/lib/useObservable';
 import * as Rx from 'rxjs';
+import { css } from '@emotion/react';
```

Comment:
@myasonik I'll defer to you on this file

Reply:
These changes are temporary:

> Remove experimental css and Emotion styles from UI components (Chrome, Canvas)

and will be removed before merging.